src/openvic-simulation/pop/Pop.hpp

```diff
@@ -99,7 +101,13 @@ namespace OpenVic {
 		fixed_point_map_t<Issue const*> PROPERTY(issue_distribution);
 		IndexedMap<CountryParty, fixed_point_t> PROPERTY(vote_distribution);
 
-		fixed_point_t PROPERTY(unemployment);
+		pop_size_t employed = 0;
+		constexpr fixed_point_t get_unemployment_fraction() {
```

Comment:
```suggestion
	public:
		constexpr fixed_point_t get_unemployment_fraction() const {
```
This needs to be `public` and `const` so it can be accessed for the Population Menu.
We'll also need a function giving the absolute number of unemployed people in the pop for the notification on the Production section of the topbar, and for calculating the number of unemployed clerks and craftsmen for the Production Menu